paint-timing/fcp-only/fcp-invisible-3d-rotate-descendant.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP due to 3d revealing</title>
+<style>
+    #main {
+        width: 100px;
+        height: 100px;
+        left: 0px;
+        position: relative;
+        top: 0;
+        transform: rotateX(45deg);
+        transform-style: preserve-3d;
+    }
+
+    #child {
```

Comment:
It would be great to have a comment: what's the purpose of the child in this test?

Reply:
ok